packages/backend/src/api/controllers/tvl/DetailedTvlController.ts

```diff
@@ -34,35 +58,45 @@ export class DetailedTvlController {
     private readonly projects: ReportProject[],
     private readonly tokens: Token[],
     private readonly logger: Logger,
+    private readonly aggregatedConfigHash: Hash256,
+    private readonly flags: DetailedControllerBehaviorFlags,
```

Comment:
Can we just call it options: DetailedTvlControllerOptions?

Reply:
Yess we can
https://github.com/l2beat/l2beat/pull/1776/commits/e5a48d7e0833e68dce0fde495a2cbe44590fd7e2